src/Acquisition.cs

```diff
@@ -305,8 +310,22 @@ public void Dispose()
             using var bayerImage = new Image((PixelFormatName)buffer.PixelFormat(), buffer.BasePtr(),
                 buffer.Size(), buffer.Width(), buffer.Height());
 
-            // Debayering and convert IDS peak IPL Image to RGB8 format
-            var bgraImage = _imageConverter.Convert(bayerImage, PixelFormat);
+            Image bgraImage;
+
+            //Apply hotpixel correction
+            if (_correctHotPixels)
+            {
+                var vec = _m_hotpixelCorrection.Detect(bayerImage);
+                var correctedImage = _m_hotpixelCorrection.Correct(bayerImage, vec);

```

Comment:
the correctedImage needs disposing, right? with a "using"?!

Reply:
fixed in 68a2a05